spec/1.1/index.bs

```diff
@@ -212,6 +212,8 @@ Passing the value returned by {{requestAnimationFrame()}} to will unregister the
 <dfn method for="VRDisplay">requestPresent()</dfn>
 Begins presenting the contents of the specified {{VRLayerInit}} array on the {{VRDisplay}} and fulfills the returned promise when presentation has begun. If {{canPresent}} is false the promise MUST be rejected. If the {{VRLayerInit}} array contains more than {{maxLayers}} elements the promise MUST be rejected. If {{requestPresent()}} is called outside of an engagement gesture the promise MUST be rejected unless the {{VRDisplay}} was already presenting. This engagement gesture is also sufficient to allow <a href="http://w3c.github.io/pointerlock/">requestPointerLock</a> calls until presentation has ended. The user agent MAY reject the promise for any other reason. If the {{VRDisplay}} is already presenting when {{requestPresent()}} is called the {{VRDisplay}} SHOULD update the {{VRLayerInit}} list being presented. If a call to {{requestPresent()}} is rejected while the {{VRDisplay}} is already presenting the {{VRDisplay}} MUST end presentation.
 
+In order for a WebGL canvas to be used as a {{VRSource}}, the context MUST be compatible with the {{VRDisplay}}. This can mean different things for different environments - for example, on a desktop computer, this means that the context MUST be created against the graphics adapter that the {{VRDisplay}} is physically plugged into. In the event that a context is not already compatible with the {{VRDisplay}}, while the requestPresent promise is outstanding, the <a href="https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.13">context will be lost and attempt to recreate itself</a> using the compatible graphics adapter. If the user agent requests user consent in response to {{requestPresent()}}, the context lost operation MUST NOT happen prior to the user granting consent. It is the webpage's responsibility to handle WebGL context loss properly, calling preventDefault() during webglcontextlost and recreating any necessary WebGL resources during the webglcontextrestored event. If the context lost event is not handled (i.e. preventDefault was not called during the firing of webglcontextlost) the {{requestPresent()}} promise MUST be rejected. If the context lost event is handled, all other WebGL contexts on the page MAY also have their context lost as byproduct of using the correct graphics adapter instead of the default one. The promise may also fail for a variety of other reasons, such as the context being actively used by a different, incompatible {{VRDisplay}}. The fulfillment of the {{requestPresent()}} promise implies that the WebGL context is now compatible with the {{VRDisplay}}.
```

Comment:
few grammatical nits:

- `context lost operation` -> `context-lost operation`
- `user granting consent` -> `user's granting consent`
- `webpage's` -> `web page's` (or `document's`)
- `context lost event` -> `context-lost event`
- `(i.e. preventDefault` -> `(i.e., preventDefault`
- `the {{requestPresent()}}` -> `, the {{requestPresent()}}`
- `context lost event is handled` -> `context-lost event is handled`